doc/source/user_guide/indexing.rst

```diff
@@ -933,6 +933,20 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+``iloc`` distinguishes between different types of boolean indexing.  If the type of boolean indexing is ``Series``,
+an error will be raised. For instance, in the following example, ``df.iloc[sr.values, 1]`` is ok. 
+This type of boolean indexing is array. But ``df.iloc[sr, 1]`` would raise ValueError.
+
+.. ipython:: python
+
+   df = pd.DataFrame([[1, 2], [3, 4], [5, 6]], list('abc'), ['column_1', 'column_2'])
```

Comment:
make the column names `'A'`, `'B'` (to make this look a bit cleaner)

Reply:
Done.